lib/puppet/application/cert.rb

```diff
@@ -145,7 +164,10 @@ def help
   needs to be restarted after revoking certificates.
 
 * sign:
-  Sign an outstanding certificate request.
+  Sign an outstanding certificate request. If '--interactive' or '-i' is
+  supplied the user will be prompted to confirm that they are signing the
+  correct certificate (recommended). If '--assume-yes' or '-y' is supplied
```

Comment:
Yes, we briefly discussed the flag pairs (`-I` `--non-interactive`), (`-y` `--assume-yes`), and (`-f` `--force`). `-I` seemed a logical negation for (`-i` `--interactive`) though we couldn't think of a CLI precedent for that particular usage. (`-f`, `--force`) is used by `cp`, `mv`, `rm`, etc for destructive commands and conflicts, while (`-y`, `--assume-yes`) is used by `apt`, `rpm`, etc for confirming the installation of packages. We asked each of the participants what they would expect to see to override the interactive behavior and IIRC they unanimously said `-y`.